CHANGELOG.md

```diff
@@ -1,11 +1,27 @@
 # Upcoming
 
+# v8.0.0-beta
+
 ## Breaking changes
 
-- (>=v8.0.0) Setting model data for a `qx.ui.table.Table` when the table is still editing will
+- Setting model data for a `qx.ui.table.Table` when the table is still editing will
 now raise an error as this could have lead to an invalid edit. To prevent any errors, ensure
 that the table edits are completed or cancelled before refreshing table model data.
 
+- Properties and members are now in the same namespace. Formerly, a class could have a member variable and a property with the same name, and there was no conflict. Now, since properties are first-class and can be manipulated as normal members, the properties and members use the same namespace, so a single name can not be defined in both.
+
+- Refining a property in a subclass used to modify it in place. It now adds it to the subclass' prototype, so it ends up in the prototype chain twice.
+
+- The predefined instance.name variable is no longer predefined because, with first-class properties, it conflicts with the commonly used property name name. Use instance.classname instead.
```

Comment:
```suggestion
- The predefined instance.name variable is no longer predefined because, with first-class properties, it conflicts with the commonly used property name "name". Use `instance.classname` instead.
```